src/skeleton/__test__/skeleton.scenario.js

```diff
@@ -0,0 +1,43 @@
+
+import * as React from 'react';
+import {Skeleton} from "../index.js";
+
+
+export default function Scenario() {
+  return (
+    <div>
+      <Skeleton
```

Comment:
Could you post a screenshot of what this looks like?

Reply:
I have replaced the old example with a new one(youtube style). And now it will look like this:
<img width="1440" alt="Screen Shot 2020-07-11 at 10 36 06 AM" src="https://user-images.githubusercontent.com/40701517/87230128-7081fd80-c362-11ea-8305-1ae1b8cff8be.png">